src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java

```diff
@@ -56,7 +56,7 @@ public MedicalInventory newMedicalInventory(MedicalInventory medicalinventory) t
 	 * @return the updated {@link MedicalInventory} object.
 	 * @throws OHServiceException
 	 */
-	public MedicalInventory updateMedicalInventoryRow(MedicalInventory medicalInventory) throws OHServiceException {
+	public MedicalInventory updateMedicalInventory(MedicalInventory medicalInventory) throws OHServiceException {
```

Comment:
Why this change if the service method called is `updateMedicalInventoryRow`?

Reply:
Because the right name is `updateMedicalInventory ` not `updateMedicalInventoryRow` we already have `updateMedicalInventoryRow` on `MedicalInventoryRowManager`

Reply:
ok so you have to change also the called method on ioOperations?